src/components/typography/Text.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { FC } from 'react';
+import styled, { StyledComponentProps, css } from 'styled-components';
+import { colors } from 'common/colors';
+import { Icon } from '../../';
+
+type TextProps = StyledComponentProps<'p', any, any, any>;
+interface QuoteProps extends StyledComponentProps<'blockquote', any, any, any> {
+  by?: string;
+}
+
+const TextStyle = css`
+  color: ${colors.grayDarken90};
+  line-height: 1.5;
+`;
+
+const StyledInfo = styled.p`
+  font-size: var(--font-size-sm);
+  ${TextStyle}
+  color: ${colors.grayDarken60};
+`;
+const StyledIngress = styled.p`
+  font-size: var(--font-size-md);
+  margin: 2rem 0;
+  font-style: italic;
+  ${TextStyle}
+`;
+const StyledParagraph = styled.p`
+  font-size: var(--font-size-md);
+  margin: 2rem 0;
+  ${TextStyle}
+`;
+const StyledQuote = styled.blockquote`
+  font-size: var(--font-size-lg);
+  font-style: italic;
+  text-align: center;
+  margin: 2rem 0;
+  ${TextStyle}
```

Comment:
We should probably avoid using too many font styles. Increasing the font and add more spacing should be sufficient enough to make Ingress and Quote "stand-out".

Referring to point 7., 8. and 12. in [0].

[0]: https://uxplanet.org/common-webpage-design-mistakes-59eed9831bd7

Reply:
I think you might be overthinking the nitpickery. When you have a full article written out, most people wouldn't mix too many different styles so close, so it wouldn't be an issue. Ingress is totally fine as is, but we can talk to Peter who does most the article writing to see if he prefers italics or larger font.
For the blockquote, I'd prefer dropping the italics, but that's mostly a personal preferance.
I can look into reviewing the rest some other time